server/images.go

```diff
@@ -36,7 +36,10 @@ import (
 
 type Capability string
 
-const CapabilityCompletion = Capability("completion")
+const (
+	CapabilityCompletion = Capability("completion")
+	CapabilitySuffix     = Capability("suffix")
```

Comment:
I like how this matches in the input variable, but wonder if we want to call it `fim` or `fill-in-the-middle` (better if one word vs dashes) or another term that might signify what the capability does

Reply:
Or perhaps `insert` https://platform.openai.com/docs/guides/text-generation/inserting-text

Reply:
I originally had this as `CapabilityInfill` but matching the input variable is really convenient